config/config.js

```diff
@@ -135,6 +135,16 @@ module.exports = {
       type: 'boolean',
       userCanEdit: false,
       adminOnly: false
+    },
+    {
+      key: 'ignoreThreatIntelligenceResults',
+      name: 'Ignore Threat Intelligence Results.',
```

Comment:
@penwoodjon Is this option named correctly? Based on the ticket it seems like the use case here is to allow turning off threat intelligence searches so if you have multiple MS sentinel instances you aren't searching for the same Threat Intel multiple times.  In looking at the implementation it seems like enabling this option turns off threat intelligence lookups.

If that is the case then I think it would be more clear to call this option something like:

"Enable Threat Intelligence Searches"
description: "If enabled, the integration will search threat intelligence indicator data"
default: true